pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.sonarsource.parent</groupId>
     <artifactId>parent</artifactId>
-    <version>70.0.0.270</version>
+    <version>71.0.0.1292</version>
```

Comment:
Double checking - was this change intended?

Reply:
Yes, because something related to licenses is coming from there. without this change build will fail